### ISSUE TITLE
A11y_SQL Azure Data Studio_Extensions - install_Screen reader: After invoking "Install" button Screen Reader is not announcing "Installation" status whether it is installed or not.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/azuredatastudio#24161]
### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Aug2023; #SQL Azure Data Studio; #WCAG4.1.3; #ScreenReader; #Win32; #Benchmark; #FTP; #STP;

### Environment Details:
Application Name: Sql Azure Data studio
Version: 1.45.1
OS: Win11

### Repro Steps:

1. Launch Azure Data Studio.
2. From the left menu, click the 'Extensions' icon to open the Extension Manager.
3. Choose any extension from the library.
4. Click the green 'Install' button to install. 
5. Now verify that NVDA/Narrator is not announcing "Installation" status.

### Actual Results:
After invoking "Installation" button Screen Reader is not announcing "Installation" status whether it is installed or not.
**Note:**
**1.Similar issue observed with chrome+ NVDA.**
**2.Similar issue is also reproducing in the below scenarios:**
**1. Issue: Extensions - Uninstall**
1. Launch Azure Data Studio.
2. From the left menu, click the 'Extensions' icon to open the Extension Manager.
3. Choose any extension from the library.
4. Click the green 'Install' button to install. Blue 'Uninstall' button should now appear.
5. Click the blue 'Uninstall' button to uninstall.
7. Now verify that NVDA/Narrator is not announcing "Uninstall" status.

**2.Issue: New Terminal**
1. Launch Azure Data Studio.
2. Navigate to the menu and click on View > Terminal
3. Click on the "Plus" sign
4. Now verify that NVDA/Narrator is not announcing whether it is created or not.

### Expected Results:
After invoking "Installation" button Screen Reader should announce "Installation" status whether it is installed or not.

### User Impact:
Screen reader user will not be able to know installation status whether it is installed or not. If NVDA/Narrator not announcing.

### Attachment:

[Install and Uninstall NVDA .webm](https://github.com/microsoft/azuredatastudio/assets/142479541/1452c977-718a-433f-a831-6a3cb17d68b6)

[install and uninstall (1).webm](https://github.com/microsoft/azuredatastudio/assets/142479541/1b5bde5c-8405-4f04-aaaf-15b07597587c)

<img width="960" alt="MicrosoftTeams-image (3)" src="https://github.com/microsoft/azuredatastudio/assets/142479541/b0174de2-ef4b-4247-b97a-07e5fecb4144">

[NVDA.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/c348b0df-6217-4925-8f4d-dc2233fc6e9c)

[Narrator.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/def3fd71-2edf-4d66-a861-19b3aa7141c7)

<img width="960" alt="MicrosoftTeams-image (1)" src="https://github.com/microsoft/azuredatastudio/assets/142479541/4bdf5e3f-1917-4e64-9dde-f74f21abe2b6">

**Comment by kisantia** [comment]
@PadminiGariki this is now fixed in the [latest insiders build](https://github.com/microsoft/azuredatastudio#download-the-latest-azure-data-studio-release). Please verify and close the issue.
![extensionInstall](https://github.com/microsoft/azuredatastudio/assets/31145923/b8017461-9c46-47ae-93ea-394d1469476f)

**Comment by PadminiGariki** [comment]
@kisantia 
verified the bug in this environment [Latest insiders build]( https://github.com/microsoft/azuredatastudio#download-the-latest-azure-data-studio-release).
Here Issue is fixed partially. 
After invoking "Plus" sign button Screen reader is announcing whether it is New Terminal created or not. This is issue is fixed.
After invoking "Install" or "Uninstall" button Screen Reader is not announcing "Installation" or "Uninstallation" status whether it is (installed/uninstallation) completed or not. This is still repro.
hence that Bug is reopened. 
Attachments is added for reference. 
#Reopened; #Regressed:08-25-23;
### Attachments:
[Narrator issue not fixed.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/1b74e149-7dcf-4675-9d72-2782205b9947)


https://github.com/microsoft/azuredatastudio/assets/142479541/b791bf0e-6680-4d93-a7bf-200e5bbe051f



[similar issue.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/6bfdac0a-2f85-4d7d-837f-ee9320cb2b46)

[NVDA similar issue (2).webm](https://github.com/microsoft/azuredatastudio/assets/142479541/81dd79e4-aeb4-4815-8e78-973e75d0247d)

**Comment by kisantia** [comment]
@PadminiGariki can you please open a separate issue for that? It is a different part of the product and unrelated to the extension installation, so it should be a different bug. The initial reported bug is fixed.

**Comment by PadminiGariki** [comment]
@kisantia As per above comment. We are closing this bug and Open new bug for that issue.
Thank you!
#closed;
